modular_skyrat/modules/mapping/code/wardrobes.dm

```diff
@@ -27,7 +27,10 @@
 					/obj/item/clothing/shoes/combat/coldres = 2)
 	premium = list(/obj/item/knife/combat/survival = 1,
 					/obj/item/storage/fancy/cigarettes/cigpack_syndicate = 5,
-					/obj/item/clothing/gloves/combat = 3)
+					/obj/item/clothing/gloves/combat = 3,
+					/obj/item/clothing/under/syndicate/skyrat/maid = 5,
+					/obj/item/clothing/gloves/combat/maid = 5,
+					/obj/item/clothing/head/maidheadband/syndicate = 5)
```

Comment:
Try to reduce the indentation of this whole list too, just, in general, formatting it like all the other lists with a ton of entries usually are. Hit me up on Discord if you're not sure.
```suggestion
					/obj/item/clothing/gloves/combat = 3,
					/obj/item/clothing/under/syndicate/skyrat/maid = 5,
					/obj/item/clothing/gloves/combat/maid = 5,
					/obj/item/clothing/head/maidheadband/syndicate = 5,
	)
```